### ISSUE TITLE
Test software appears to be out of date.

### ISSUE THREAD

**Issue by droshux** [Dyalog/ride#1231]
Reading through `package.json`, I found that the Spectron version used is far behind appropriate for the electron version used, especially since *Sprectron was deprecated on the 1st of February 2022*. This seems to mean that the RIDE hasn't been tested in over two years since when running `npm run test`, an error gets spat out that seams to lead to this conclusion: *session not created: This version of ChromeDriver only supports Chrome version 98*, which led me to [this issue](https://github.com/electron-userland/spectron/issues/1046).  This seems totally ridiculous and so I thought I'd check in before totally re-doing all the tests. 

This is more likely an issue on my end and if it is, how would I resolve it, (more contributor info would be great!).
The browsers I have installed are: `Mozilla FIrefox 126.0.1` and `Vivaldi 6.7.3329.39 stable` (chromium based) in case this info is necessary. 

To recreate: git clone the repo, `npm i` and `npm run test`.

Sorry if this is a waste of time.

**Comment by e9gille** [comment]
You are not mistaken, the test framework is broken and the tests haven't been executed for some time. The test code was left in place pending a replacement of the framework, but that could/should have been made clear.
